### DISCUSSION TITLE
Outlook Add-on: listen for subject changed on compose mode.

### DISCUSSION THREAD

**Discussion by joaquinlefebvre** [OfficeDev/office-js#4097] [General]
Hi all!

We are developing an add-in for Outlook and in compose mode we correctly receive the changes in both the attachments and the recipients but we cannot obtain information about which Office.EventType to listen to to detect the change in the subject.

Is there any other mechanism or can't it be done?

**Comment by sibyavarachan89** [comment]
Have you found a solution? I have the same requirement, but I could not find any subject change event.
